### PR TITLE
Add forcats_tutorial_translation.rmd

### PR DESCRIPTION
**Pull request by xiaoyunzoezhu**
This is our community contribution file

### CODE REVIEW

File: forcats_tutorial_translation.Rmd [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Forcats Tutorial Translation
+
+Xu Xu and Xiaoyun Zhu
+
+
+```{r setup, include=FALSE}
```
Comment (st3186) [inline forcats_tutorial_translation.Rmd]: this should read {r, include=FALSE}. There should be no "setup" tag in your submission

**Review by st3186 — CHANGES_REQUESTED** [review]
Looks good, except for the named `setup` tag.  Also, you've jumped straight from # -> ###;  These will be rendered as numerical headers (i.e. going straight from 4 to 4.1.1), so it may be better to use '##' instead of '###'. Please fix and revert to us, thanks!

File: forcats_tutorial_translation.Rmd (hunk 1 above)
Comment (xiaoyunzoezhu) [reply forcats_tutorial_translation.Rmd]: We have fixed the issues above and committed it again.

**Comment by st3186** [comment]
Thanks, changes are received and we can proceed with trying to merge!

**Comment by bojum** [comment]
🎉 Merged. Thanks for your contribution!

**Pull request merged**